html-css/week1/nadiia/portfolio/index.html

```diff
@@ -0,0 +1,298 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+    />
+    <link href="styles.css" rel="stylesheet" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav>
+      <a href="#about">About myself</a>
+      <a href="#project">Projects</a>
+      <a href="#art">Artworks</a>
+      <a href="#contact">Contact me</a>
+    </nav>
```

Comment:
```suggestion
<nav aria-label="Main navigation">
  <ul>
    <li><a href="#about">About myself</a></li>
    <li><a href="#project">Projects</a></li>
    <li><a href="#art">Artworks</a></li>
    <li><a href="#contact">Contact me</a></li>
  </ul>
</nav>
```

There are a couple of things that happen here. First, we've added an `aria` attribute that identifies to screenreaders that this is the main navigation on the website. This is very helpful, considering there can be multiple `<nav>` tags on a page.
Second - we create an unordered list and wrap each item in a `<li>` tag. This strengthens the associations of all the `<a>` tags as part of `<nav>`. This helps again screen readers to indicate how many items are in the list, and there is a special navigation function to go through lists in particular. You can read more in this summary on an old discussion if `<a>` tags should be wrapped or not - https://css-tricks.com/wrapup-of-navigation-in-lists/